docs/pages/modules/get-started.mdx

```diff
@@ -4,19 +4,128 @@ sidebar_title: Get started
 description: Learn about getting started with Expo modules API.
 ---
 
+import { BookOpen02Icon, Grid01Icon } from '@expo/styleguide-icons';
+
 import { BoxLink } from '~/ui/components/BoxLink';
+import { Step } from '~/ui/components/Step';
 import { Terminal } from '~/ui/components/Snippet';
 
+If you need to extend the capabilities of your application, you can install expo packages or third party react native libraries.
+Sometimes, this is not enough. You may need to write custom native code to tap into obscure platform APIs or use custom Android or iOS dependencies.
+
+Luckily, Expo has first class support for this in the form of Expo Modules.
+
+Expo Modules allow you to write native code in a way that feels natural with minimal boilerplate that is also consistent on both platforms.
+
+<BoxLink
+  title="Expo Modules Overview"
+  description="Learn more about the design considerations behind Expo Modules."
+  href="./overview"
+  Icon={BookOpen02Icon}
+/>
+<BoxLink
+  title="Expo Modules API Reference"
+  description="A list of building blocks you can use to create your own Expo Module."
+  href="./module-api"
+  Icon={Grid01Icon}
+/>
+
 There are two ways to get started with the Expo Module API: you can either initialize a new module from scratch or add the Expo Module API to an existing module. This guide will walk you through creating a new module from scratch, and the [Integrating in an existing library guide](/modules/existing-library) covers the latter case.
 
-## 1. Creating a new module
+We provide two recommended flows for creating a new Expo Module. You can either:
+
+- add a new module to an existing Expo application and use it to test and develop your module
+- create a new module in isolation with a generated example project, if you want to reuse it in multiple projects or publish to npm.
+
+The next two sections describe the two flows in detail.
+
+## Adding a new module to an existing application
+
+<Step label="1">
+### Creating the local Expo Module
```

Comment:
```suggestion
<Step label="1">

### Creating the local Expo Module
```

iirc, there should be an extra line between section title and opening and closing `<Step>` components otherwise it might not render correctly.